code/modules/shuttle/shuttle.dm

```diff
@@ -21,8 +21,6 @@
 	var/dwidth = 0	//position relative to covered area, perpendicular to dir
 	var/dheight = 0	//position relative to covered area, parallel to dir
 
-	shuttle_abstract_movable = 1
```

Comment:
You dont need this here, this would make it so stationary docks move. You added shuttle_abstract_movable to mobile ports already like I asked.